### ISSUE TITLE
Fresh install on Mojolicious 9.07 is failing

### ISSUE THREAD

**Issue by aimass** [jhthorsen/mojolicious-plugin-openapi#202]
Hi there!

Installing: Mojolicious-Plugin-OpenAPI-4.00
On 5.30.3 inside a Docker container

Results in (picking the most relevant errors), seem all related to Mojo/Util.pm line 498.

```
Checking dependencies from MYMETA.json ...
Checking if you have JSON::Validator 4.13 ... Yes (4.14)
Checking if you have Test::More 0.88 ... Yes (1.302162)
Checking if you have Mojolicious 9.00 ... Yes (9.07)
Building and testing Mojolicious-Plugin-OpenAPI-4.00
[...]
t/00-basic.t ........................... ok
Use of uninitialized value $_[0] in hash element at /usr/local/lib/perl5/site_perl/5.30.3/Mojo/Util.pm line 498.
    # No tests run!

#   Failed test 'No tests run for subtest "/v2"'
#   at t/basic-404-501.t line 30.
Can't call method "get_ok" on an undefined value at t/basic-404-501.t line 26.
[...]
t/basic-bundle.t ....................... ok
Use of uninitialized value $_[0] in hash element at /usr/local/lib/perl5/site_perl/5.30.3/Mojo/Util.pm line 498.
Can't call method "app" on an undefined value at t/basic-coerce.t line 21.
t/basic-coerce.t ....................... 
[...]
t/v2-file.t ............................ ok
Format rule for 'unknown' is missing at /usr/local/lib/perl5/site_perl/5.30.3/JSON/Validator.pm line 674.
t/v2-formats.t ......................... ok
```

We'll keep trying to figure out the root cause but publishing here just in case someone else is having the same issue.

**Comment by jberger** [comment]
I believe this has been fixed in Mojolicious 9.08 (ref: https://github.com/mojolicious/mojo/issues/1735). Please reopen if that is not the case. Thank you.

**Comment by aimass** [comment]
Yep it did! Thank you !!
